inception-workload-dynamic/src/main/java/de/tudarmstadt/ukp/inception/workload/dynamic/page/workload/DynamicWorkloadManagementPage.java

```diff
@@ -98,19 +96,24 @@
     public DynamicWorkloadManagementPage() {
         super();
         //Error, user is returned to home page, nothing else to do
-        error("No Project selected, please enter the monitoring page only with a valid project reference");
         setResponsePage(getApplication().getHomePage());
+        getSession().error("No project selected.");
     }
 
     //Constructor with a project
-    public DynamicWorkloadManagementPage(final PageParameters aPageParameters)
-    {
+    public DynamicWorkloadManagementPage(final PageParameters aPageParameters) {
         super(aPageParameters);
 
         //Get current Project
         currentProject.setObject(getProjectFromParameters(aPageParameters.get
             (PAGE_PARAM_PROJECT_ID)).get());
 
+        init();
+    }
+
+    public void init()
```

Comment:
You can probably use/override `onInitialize()` instead: https://ci.apache.org/projects/wicket/guide/6.x/guide/componentLifecycle.html#componentLifecycle_3